CONTRIBUTING.md

```diff
@@ -36,20 +36,31 @@ Demos and examples 🤓 can be found on [fabricjs.com](http://fabricjs.com/demos
 
 **These are minimal requirements. Without them issues shall be ⛔.**
 
-If it's not a bug **OR** if you're unsure, start a 🤠 [discussion](https://github.com/fabricjs/fabric.js/discussions) or create a post ✉️ on [Fabric's google group](groups.google.com/forum/?fromgroups#!forum/fabricjs) where someone might clarify some of the things.
+If it's not a bug **OR** if you're unsure, start a 🤠 [discussion](https://github.com/fabricjs/fabric.js/discussions) or create a post ✉️ on [Fabric's google group](https://groups.google.com/g/fabricjs) where someone might clarify some of the things.
```

Comment:
I still think we should remove references to the google group, github discussions is a much better forum for q&a

Reply:
I am for

Reply:
yes kill the google group from the docs.
Let's clarify we don't actively spend time on it